ephios/extra/templates/extra/widgets/recurrence_picker.html

```diff
@@ -0,0 +1,231 @@
+{% load i18n %}
+{% load static %}
+
+<input type="hidden" autocomplete="off" id="pick_hour" value="{{ widget.pick_hour|lower }}">
+<div id="vue_app" class="row mt-4">
+    <div id="chooser" class="col-6">
+        <div class="input-group pb-4">
+            <div class="input-group-text">{% translate "starting at" %}</div>
+            <input type="date" class="form-control" v-model="DTSTART">
+            <input type="time" class="form-control" v-if="pickHour" v-model="DTSTART_TIME">
+        </div>
+        <ul class="list-group mb-4">
+            <li class="list-group-item p-3" v-for="rule in rules"
+                :class="{'border-danger': !isRuleValid(rule)}">
+                <div class="btn-toolbar justify-content-between">
+                    <div class="btn-group pb-4">
+                        <button type="button" class="btn btn-outline-secondary"
+                                :class="{'active': rule.freq===4}" v-if="pickHour"
+                                @click="changeFreq(rule, frequencies.HOURLY)">{% translate "Hourly" %}</button>
+                        <button type="button" class="btn btn-outline-secondary"
+                                :class="{'active': rule.freq===3}"
+                                @click="changeFreq(rule, frequencies.DAILY)">{% translate "Daily" %}</button>
+                        <button type="button" class="btn btn-outline-secondary"
+                                :class="{'active': rule.freq===2}"
+                                @click="changeFreq(rule, frequencies.WEEKLY)">{% translate "Weekly" %}</button>
+                        <button type="button" class="btn btn-outline-secondary"
+                                :class="{'active': rule.freq===1}"
+                                @click="changeFreq(rule, frequencies.MONTHLY)">{% translate "Monthly" %}</button>
+                        <button type="button" class="btn btn-outline-secondary"
+                                :class="{'active': rule.freq===0}"
+                                @click="changeFreq(rule, frequencies.YEARLY)">{% translate "Yearly" %}</button>
```

Comment:
`btn-outline-secondary` is a very light color. It lacks contrast and thus fails [WCAG guidelines](https://developer.mozilla.org/en-US/docs/Web/Accessibility/Understanding_WCAG/Perceivable/Color_contrast). In my opinion, it rather looks disabled than inactive.

The default bootstrap colors have much more contrast, but I don't know which other of your views might look weird if you make `secondary` darker.

Ephios:
![Screenshot of the frequency buttons](https://github.com/user-attachments/assets/406ad6a6-9553-423c-a237-70f5d34fb697)

Bootstrap default colors:
![image](https://github.com/user-attachments/assets/b881efe8-fb02-45eb-8ffd-6ed0329ff80b)

Reply:
well, basically every other view as the whole design is based on the bootstrap colors, so changing that is not really feasible.
Using primary instead seems to be too bold IMO

Reply:
I think we should take another look at this with the next bootstrap update. Many components expect secondary to have more contrast. For the places where we do need less, we can add another variant between it and `light`. Then it would also be themeable (think dark theme)

Reply:
@felixrindt so keep it as is for this PR?